Sources/MongoSwift/BSON/BSONDecoder.swift

```diff
@@ -842,15 +842,15 @@ internal struct _BSONKey: CodingKey {
     internal static let `super` = _BSONKey(stringValue: "super")!
 }
 
-internal extension DecodingError {
-    internal static func _typeMismatch(at path: [CodingKey],
+extension DecodingError {
+    static func _typeMismatch(at path: [CodingKey],
```

Comment:
I think to satisfy both SwiftLint and the compiler we still need the `internal`s on the functions, but you can leave it off of the extension.
(putting it back on the functions should also fix the `vertical_parameter_alignment` violations on Travis.)